[Gripper] New version (approche contrôlée à tester)

ValentinVERGEZ opened the pull request:
J'ouvre cette PR pour revue puis pour intégration.
Je m'occupe de tester la partie approche finale contrôlée.
@tdanel tu as de quoi valider la partie Gripper ? Elle est p-e déjà validée ?
Ou alors on l’intègre déjà considérant que ça fait partie du travail à faire par la suite ?
A vous de me dire.

<!-- Reviewable:start -->

---

This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/pyroteam/robocup-pkg/124)

<!-- Reviewable:end -->

tdanel committed:
Modification du noeud et des messages pour piloter les deux servos

tdanel committed:
conception en cours

tdanel committed:
Ajout pilotage 2 servos

tdanel committed:
dernier commit avant la fin du monde

tdanel committed:
qq

tdanel committed:
Noeud gripper codé non compilé et sans l'action a l'approche finale

tdanel committed:
Merge remote-tracking branch 'origin/robocup-devel' into work/thomas/hardware/newGripper

tdanel committed:
en cours de modifs

tdanel committed:
Avant merge

tdanel committed:
Merge remote-tracking branch 'origin/robocup-devel' into work/thomas/hardware/newGripper

Conflicts:
	hardware/gripper/src/gripper_node.cpp

tdanel committed:
Gripper node compile

TODO: Noeud sur arduino à vérifier

ValentinVERGEZ committed:
Merge remote-tracking branch 'origin/robocup-devel' into work/thomas/hardware/newGripper

ValentinVERGEZ committed:
Ajout code de gestion de Controlled By Param

tdanel committed:
Version fonctionnelle du gripper node avec rosserial_arduino sans approche finale

roscore
rosrun rosserial_python serial_node.py _port:=/dev/ttyACM*
rosrun gripper gripper_node

pour prendre :

rosservice call /hardware/high_level/gripper_srv "cmd: 1"

pour déposer :

rosservice call /hardware/high_level/gripper_srv "cmd: 2"

tdanel committed:
Merge branch 'work/thomas/hardware/newGripper' of https://github.com/PyroTeam/robocup-pkg into work/thomas/hardware/newGripper

tdanel commented:
La partie gripper tout seul fonctionne, avec appel de service au noeud gripper node
Le fonctionnement a été mis dans un commit (7cddc38), peut-être à changer en README ^^

vcoelen commented:
Review status: 0 of 19 files reviewed at latest revision, 1 unresolved discussion.

---

_[hardware/gripper/firmware/arduinoGripper/arduinoGripper.ino, line 1 [r3]](https://reviewable.io:443/reviews/pyroteam/robocup-pkg/124#-KMtAjWznvY59VRpa9N4:-KMtAjWznvY59VRpa9N5:1057338350) ([raw file](https://github.com/pyroteam/robocup-pkg/blob/292ceb98c8a204e499c1ce6e10130a2466b3c5a3/hardware/gripper/firmware/arduinoGripper/arduinoGripper.ino#L1)):_

> ``` Arduino
> #if (ARDUINO >= 100)
> ```

Il y a deux arduinoGripper.ino dans deux dossiers différents? 

---

_Comments from [Reviewable](https://reviewable.io:443/reviews/pyroteam/robocup-pkg/124)_

<!-- Sent from Reviewable.io -->

tdanel commented:
Review status: 0 of 19 files reviewed at latest revision, 1 unresolved discussion.

---

_[hardware/gripper/firmware/arduinoGripper/arduinoGripper.ino, line 1 [r3]](https://reviewable.io:443/reviews/pyroteam/robocup-pkg/124#-KMtAjWznvY59VRpa9N4:-KMtdbvwrYJ59wXC90S9:-1545070954) ([raw file](https://github.com/pyroteam/robocup-pkg/blob/292ceb98c8a204e499c1ce6e10130a2466b3c5a3/hardware/gripper/firmware/arduinoGripper/arduinoGripper.ino#L1)):_
<details><summary>

<i>Previously, vcoelen wrote…</i></summary>



> Il y a deux arduinoGripper.ino dans deux dossiers différents? 
> </details>
> Je pense que ce sont les mêmes ^^ à vérifier
> Le truc c'est que l'IDE crée à chaque fois un "sur" dossier du même nom donc firmware ça allait pas

---

_Comments from [Reviewable](https://reviewable.io:443/reviews/pyroteam/robocup-pkg/124)_

<!-- Sent from Reviewable.io -->

tdanel committed:
Merge branch 'work/thomas/hardware/newGripper' of https://github.com/PyroTeam/robocup-pkg into work/thomas/hardware/newGripper

tdanel committed:
Rédaction du README
Petit changement dans le process de dépose (remarque de Vincent sur le tapis qui détecte la présence de l'objet à déposer)
L'appel à l'approche contrôlée est commenté
